index.html

```diff
@@ -1,49 +1,60 @@
-  			</h4>
-        <table class="tocTable">
-          <tr>
-            <td style="text-align: left;">Angular Pt. 1: SPA Day</td>
-            <td class="disabled-link">slides</td>
-            <td class="disabled-link">worksheet</td>
-            <!-- <td><a href="angular-series/01/single-page-apps.html">slides</a></td>
+      <br />
+      <h4>2021 Coding &amp; Cocktails Angular Series</h4>
+      <table class="tocTable">
+        <tr>
+          <td style="text-align: left">Angular Pt. 1: SPA Day</td>
+          <td class="disabled-link">slides</td>
+          <td class="disabled-link">worksheet</td>
+          <!-- <td><a href="angular-series/01/single-page-apps.html">slides</a></td>
             <td><a href="http://bit.ly/cnchtmlwork" target="_blank">worksheet</a></td> -->
-          </tr>
-          <tr>
-            <td style="text-align: left;">Angular Pt. 2: Playing with Blocks</td>
-            <td class="disabled-link">slides</td>
-            <td class="disabled-link">worksheet</td>
-            <!-- <td><a href="02-garnish/css.html">slides</a></td>
+        </tr>
+        <tr>
+          <td style="text-align: left">Angular Pt. 2: Playing with Blocks</td>
+          <td class="disabled-link">slides</td>
+          <td class="disabled-link">worksheet</td>
+          <!-- <td><a href="02-garnish/css.html">slides</a></td>
             <td><a href="http://bit.ly/cnccsswork" target="_blank">worksheet</a></td> -->
-          </tr>
-          <tr>
-            <td style="text-align: left;">Angular Pt. 3: Fake It ’Til You Make It </td>
-            <td class="disabled-link">slides</td>
-            <td class="disabled-link">worksheet</td>
-            <!-- <td><a href="04-liquor/javascript-introduction.html">slides</a></td>
+        </tr>
+        <tr>
+          <td style="text-align: left">
+            Angular Pt. 3: Fake It ’Til You Make It
+          </td>
+          <td class="disabled-link">slides</td>
+          <td class="disabled-link">worksheet</td>
+          <!-- <td><a href="04-liquor/javascript-introduction.html">slides</a></td>
             <td><a href="http://bit.ly/cncjavascriptwork">worksheet</a></td> -->
-          </tr>
-          <tr>
-            <td style="text-align: left;">Angular Pt. 4: Go Fetch</td>
-            <td class="disabled-link">slides</td>
-            <td class="disabled-link">worksheet</td>
-            <!-- <td><a href="05-enjoy/code-jam.html">slides</a></td>
+        </tr>
+        <tr>
+          <td style="text-align: left">Angular Pt. 4: Go Fetch</td>
+          <td class="disabled-link">slides</td>
+          <td class="disabled-link">worksheet</td>
+          <!-- <td><a href="05-enjoy/code-jam.html">slides</a></td>
             <td><a href="http://bit.ly/cncenjoywork">worksheet</a></td> -->
-          </tr>
-        </table>
-        <br>
-      </div>
-	</body>
+        </tr>
+      </table>
+      <br />
+      <h4>2022 Coding &amp; Cocktails CSS Series</h4>
```

Comment:
@alisaduncan @Silfron @ashbshaw This is the part im most unsure about,  I couldnt figure out how to see the slides while developing other than adding them here.  I assume this is how i will also pass the slides to Ashley for review but not 100% sure.

Reply:
@amrunnells nope, you don't need it here. What you'd do is send a link directly to the CSS index html. That's the link you'd also provide for review (and put on the cocktails-site repo). You should be able to serve this folder and navigate to `css-series/css-series-01.html`. if you have any questions feel free to post on Slack (i'll see notifications better)